make in debian testing

make
cc -g -Wall -Wextra -Werror -Wno-unused-parameter -Wno-sign-compare -Wno-unused-function -Wno-unused-variable -Wdeclaration-after-statement -DXWAYLAND -I. -DWLR_USE_UNSTABLE -std=c99 -I/usr/include/libdrm -I/usr/include/pixman-1       -c -o dwl.o dwl.c
dwl.c: In function ‘keypress’:
dwl.c:1310:19: error: comparison between ‘enum wlr_key_state’ and ‘enum wl_keyboard_key_state’ [-Werror=enum-compare]
 1310 |  if (event->state == WL_KEYBOARD_KEY_STATE_PRESSED)
      |                   ^~
cc1: all warnings being treated as errors
make: *** [<builtin>: dwl.o] Error 1

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ Feature *non-goals* include:
 
 ## Building dwl
 
-dwl has only two dependencies: wlroots 0.12 and wayland-protocols. Simply install these and run `sudo make install`.
+dwl has only two dependencies: wlroots-git and wayland-protocols. Simply install these and run `sudo make install`.
 
 To enable XWayland, you should also install xorg-xwayland and uncomment its flag in `config.mk`.
 

